openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -728,7 +728,9 @@ A Client makes a Credential Request to the Credential Endpoint by sending the fo
 
 The `proof_type` parameter is an extension point that enables the use of different types of proofs for different cryptographic schemes.
 
-The `proof` element MUST incorporate the Credential Issuer Identifier (audience), and a `c_nonce` value generated by the Authorization Server or the Credential Issuer to allow the Credential Issuer to detect replay. The way that data is incorporated depends on the key proof type. In a JWT, for example, the `c_nonce` value is conveyed in the `nonce` claim, whereas the audience is conveyed in the `aud` claim. In a Linked Data proof, for example, the `c_nonce` is included as the `challenge` element in the key proof object and the Credential Issuer (the intended audience) is included as the `domain` element.
+The `proof` element MUST incorporate the Credential Issuer Identifier (audience), and a `c_nonce` value generated by the Authorization Server or the Credential Issuer to allow the Credential Issuer to detect replay.  The Wallet MUST use this nonce for subsequent requests until a new nonce is issued by the Credential Issuer. Each issued `c_nonce` value MUST be unique, a Credential Issuer cannot issue duplicate `c_nonces` values.
```

Comment:
I think this is the wrong place in the spec to have this text - this section is about credential requests. A requirement for the wallet to use the provided value seems like it should be in the section where the value is provided.

(And I agree with the other comments, I think the 'MUST be unique' is a separate discussion that we should discuss on a issue first so we can have a proper discussion on whether it needs to be unique or not.)